.changeset/warm-zoos-fry.md

```diff
@@ -0,0 +1,12 @@
+---
+"@nextui-org/accordion": major
+"@nextui-org/use-aria-accordion-item": major
+"@nextui-org/modal": major
+"@nextui-org/navbar": major
+"@nextui-org/popover": major
+"@nextui-org/ripple": major
+"@nextui-org/tabs": major
+"@nextui-org/tooltip": major
```

Comment:
This PR doesn't include any breaking changes. I believe you can just use `patch` instead of `major`.

Reply:
Yea I thought so too buy pnpm changeset made them all major.

Reply:
You can press enter to skip. You should see `No items were selected`. For patch, just simply press enter twice. e.g.

<img width="622" alt="image" src="https://github.com/nextui-org/nextui/assets/35857179/9accff67-38fe-4267-8ad8-d94427e3dcb0">

Alternatively, you can manually update `.changeset/warm-zoos-fry.md`

Reply:
Yea I edited it should be good now hopefully.